src/http.cpp

```diff
@@ -660,6 +656,47 @@ bool HttpReqUL::prepare(FileAccess* fa, const char* tempurl, SymmCipher* key,
     // unpad for POSTing
     out->resize(size);
 
+
+    const char *data = out->data();
+    byte c[12];
```

Comment:
Better to define the `crcSize` as a #define or constant and change it below when used, instead of 12.